MSAL/test/app/mac/MSALAcquireTokenViewController.m

```diff
@@ -467,4 +524,54 @@ - (MSALAccount *)selectedAccount
     return self.accounts[self.userPopup.indexOfSelectedItem-1];
 }
 
+- (MSALInteractiveTokenParameters *)tokenParams:(BOOL)isSSOSeedingCall
```

Comment:
nit: let's call this tokenParamsWithSSOSeeding for better readability?

Reply:
Yes that's better. Thank you, I will address in next commit